### ISSUE TITLE
css/css-scroll-snap/unreachable-snap-positions-003.html fails at a 800x600 viewport

### ISSUE THREAD

**Issue by gsnedders** [web-platform-tests/wpt#48968]
As documented, testharness tests run with a viewport _at least_ 800x600.

However, this test fails when run at an 800x600 viewport, thus Safari's failure on wpt.fyi.

cc @hiikezoe (who committed the test) and @dholbert (who is credited in the commit message for originally writing it)

See also: https://github.com/web-platform-tests/rfcs/issues/188

**Comment by hiikezoe** [comment]
Yeah, indeed the test fails on such narrow width viewport. Just wondering the minimum viewport size is for desktop, right?

On mobile the test should just work because there's `<meta name="viewport" content="width=device-width,initial-scale=0.5">`.  I am wondering the minimum viewport size is realistic on desktops.

**Comment by dholbert** [comment]
> this test fails when run at an 800x600 viewport,

Yup, confirmed in Firefox (not-in-responsive-design-mode which triggers the meta-viewport scaling).
Here's a live link to the test, for convenience: https://wpt.live/css/css-scroll-snap/unreachable-snap-positions-003.html

> cc @hiikezoe (who committed the test) and @dholbert (who is credited in the commit message for originally writing it)

I don't think there's anything magic/important about the magnitudes of the lengths in the test ( `max-width: 800px`, `width: 500px;`, `610px`, etc.) -- those were just from me reducing-the-original-bug-report to something that worked in my own local browser.

Seems like we could probably reduce the widths in the testcase, so that we don't overflow out of a 600px-wide test viewport.  I just tried cutting all the horizontal sizes in half, and the test still passes in Firefox (and fails in a build before the fix landed).  I'll file a bug to make that change, and also to do `s/max-width/width/` on the scroller so we're less at-the-mercy of the viewport size.

**Comment by dholbert** [comment]
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1929309 with a patch that fixes this, I think.

**Comment by dholbert** [comment]
This should be fixed by https://github.com/web-platform-tests/wpt/pull/48994 .
